tests/Testcontainers.WebDriver.Tests/WebDriverContainerTest.cs

```diff
@@ -0,0 +1,73 @@
+namespace Testcontainers.WebDriver;
+
+[UsedImplicitly]
+public sealed class WebDriverContainerTest : IAsyncLifetime
+{
+    private readonly WebDriverContainer _webDriverContainer = new WebDriverBuilder().WithRecording().Build();
+
+    public Task InitializeAsync()
+    {
+        return _webDriverContainer.StartAsync();
+    }
+
+    public Task DisposeAsync()
+    {
+        return _webDriverContainer.DisposeAsync().AsTask();
+    }
+
+    public sealed class HelloWorldContainer : IClassFixture<WebDriverContainerTest>, IAsyncLifetime
+    {
+        private const string HelloWorldImage = "testcontainers/helloworld:1.1.0";
+
+        private readonly Uri _helloWorldEndpoint = new UriBuilder(Uri.UriSchemeHttp, "hello-world-container", 8080).Uri;
+
+        private readonly IContainer _helloWorldContainer;
+
+        private readonly WebDriverContainerTest _fixture;
+
+        public HelloWorldContainer(WebDriverContainerTest fixture)
+        {
+            _fixture = fixture;
+
+            // TODO: Pass the depended container (Docker resource) to the builder and resolve the dependency graph internal.
+            _helloWorldContainer = new ContainerBuilder()
+                .WithImage(HelloWorldImage)
+                .WithNetwork(_fixture._webDriverContainer.GetNetwork())
+                .WithNetworkAliases(_helloWorldEndpoint.Host)
+                .WithPortBinding(_helloWorldEndpoint.Port, true)
+                .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(request =>
+                    request.ForPath("/").ForPort(Convert.ToUInt16(_helloWorldEndpoint.Port))))
+                .Build();
+        }
+
+        public Task InitializeAsync()
+        {
+            return _helloWorldContainer.StartAsync();
+        }
+
+        public Task DisposeAsync()
+        {
+            return _helloWorldContainer.DisposeAsync().AsTask();
+        }
+
+        [Fact]
+        public async Task HeadingElementReturnsHelloWorld()
+        {
+            // Given
+            using var driver = new RemoteWebDriver(new Uri(_fixture._webDriverContainer.GetConnectionString()), new ChromeOptions());
+
+            // When
+            driver.Navigate().GoToUrl(_helloWorldEndpoint.ToString());
+            var headingElementText = driver.FindElementByTagName("h1").Text;
+
+            await _fixture._webDriverContainer.StopAsync()
+                .ConfigureAwait(false);
+
+            await _fixture._webDriverContainer.ExportVideo(Path.Combine(TestSession.TempDirectoryPath, Path.GetRandomFileName()))
+                .ConfigureAwait(false);
```

Comment:
@kiview @eddumelendez to read the video from the FFmpeg container, it is necessary to stop it first. How does the VNC screen recording container does it?

Reply:
It could be that the `selenium/video` image works differently than `testcontainers/vnc-recorded`. Given that tc-java is using `testcontainers/vnc-recorded`, it doesn't need to stop in order to stop the container.

Reply:
@eddumelendez 
How do you get updated about the testcontainers/vnc-recorded` image? 
If is there a change in the configuration or env? 
I prepare to be depend only on selenium